include/itkButterworthBandpass1DFilterFunction.h

```diff
@@ -0,0 +1,93 @@
+/*=========================================================================
+ *
+ *  Copyright Insight Software Consortium*
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *=========================================================================*/
+#ifndef itkButterworthBandpass1DFilterFunction_h
+#define itkButterworthBandpass1DFilterFunction_h
+
+#include "itkFrequencyDomain1DFilterFunction.h"
+#include "itkMath.h"
+
+namespace itk
+{
+class ButterworthBandpass1DFilterFunction:
+  public FrequencyDomain1DFilterFunction
+{
+public:
+  /** Standard class typedefs. */
+
+  typedef ButterworthBandpass1DFilterFunction    Self;
+  typedef itk::FrequencyDomain1DFilterFunction   Superclass;
+  typedef itk::SmartPointer< Self >              Pointer;
+  typedef itk::SmartPointer< const Self >        ConstPointer;
```

Comment:
style nitpick: remove `itk::` when inside the `itk` namespace (i.e. the `namespace itk` is on line 24`)